src/DotLiquid.Tests/Tags/Html/TableRowTests.cs

```diff
@@ -19,6 +19,15 @@ public void TestHtmlTable()
                 Hash.FromAnonymousObject(new { numbers = new int[] { } }));
         }
 
+        [Test]
+        public void TestHtmlTableWithDefaultCols()
+        {
+            Helper.AssertTemplateResult(
+                string.Format("<tr class=\"row1\">{0}<td class=\"col1\"> 1 </td><td class=\"col2\"> 2 </td><td class=\"col3\"> 3 </td></tr>{0}", Environment.NewLine),

```

Comment:
nit: @daviburg won't like this - methods with multiple of same type should use named parameters.

Reply:
Just suggesting that for clarity the named parameters assert for humans the match been inputs when compiler cannot verify it.

Reply:
This document explains what named parameters are, with examples: https://learn.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments#named-arguments

Reply:
sorry for my bad in interpreting your comment, and bad in my english. I updated new change